ui/app/serializers/config-ui/message.js

```diff
@@ -3,22 +3,37 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { encodeString } from 'core/utils/b64';
+import { decodeString, encodeString } from 'core/utils/b64';
 import ApplicationSerializer from '../application';
 
 export default class MessageSerializer extends ApplicationSerializer {
   primaryKey = 'id';
```

Comment:
primaryKey is id by default—see [docs](https://guides.emberjs.com/release/models/customizing-serializers/#toc_ids). You can probably remove and add a comment that the backend returns id.